sdk/eventgrid/tests.yml

```diff
@@ -4,5 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: eventgrid
-    EnvVars:
-      AZURE_EVENTGRID_TEST_MODE: Live
+    SDKType: client
+    TimeoutInMinutes: 240
```

Comment:
I'm not sure we actually need any of these changes and I'd suggest undoing all the changes except for the SDKType addition for this file. I would prefer the library owner do any specific timeout or cloud configuration changes so they can be validated first.

Reply:
Sure, revert except the `EnvVars` which is replaced by SDKType